CHANGELOG.md

```diff
@@ -6,26 +6,37 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
+This release contains new features and bug fixes since the 2.9.3 release.
+We deem it moderate priority for upgrading.
+
 **Features**
+* #4874 Allow joins in continuous aggregates
```

Comment:
Isn't this considered noteworthy feature?

Reply:
Another thing to consider @iroussos is about full support for PG15, I guess this release we do it, right?

Reply:
would this change be fine?

This release includes these noteworthy features:
* Joins in continuous aggregates